Sources/Testing/Expectations/Expectation.swift

```diff
@@ -10,9 +10,9 @@
 
 /// A type describing an expectation that has been evaluated.
 public struct Expectation: Sendable {
-  /// The source code of the expression evaluated by this expectation, if
-  /// available at compile time.
-  public var sourceCode: SourceCode?
+  /// The expression evaluated by this expectation, if available at compile
+  /// time.
+  public var evaluatedExpression: Expression?
```

Comment:
Could the `evaluatedExpression` property become non-`Optional` now?

Reply:
Maybe—I'm honestly not sure if there are outstanding code paths that don't produce a value here… I'll investigate.

Reply:
Works!